WcaOnRails/spec/mailers/competitions_mailer_spec.rb

```diff
@@ -186,16 +186,80 @@
+      end
+
+      it "renders the headers" do
+        expect(mail.subject).to eq "[wca-report] [Oceania] Comp of the Future 2016"
+        expect(mail.to).to eq ["reports@worldcubeassociation.org"]
+        expect(mail.cc).to match_array competition.delegates.pluck(:email) + ["regulations@worldcubeassociation.org"] + ["disciplinary@worldcubeassociation.org"]
+        expect(mail.from).to eq ["notifications@worldcubeassociation.org"]
+        expect(mail.reply_to).to match_array competition.delegates.pluck(:email)
+      end
+
+      it "renders the body" do
+        expect(mail.body.encoded).to match(/@WRC: Feedback requested on incidents: 1, 2, 3/)
+        expect(mail.body.encoded).to match(/@WDC: Feedback requested on incidents: 4, 5, 6/)
+        expect(mail.body.encoded).to match(/This was a great competition/)
+      end
     end
 
-    it "renders the body" do
-      expect(mail.body.encoded).to match(/This was a great competition/)
+    context "wdc feedback requested" do
+      before(:each) do
+        competition.delegate_report.update_attributes!(wdc_feedback_requested: true, wdc_incidents: "4, 5, 6")
+      end
+
+      it "renders the headers" do
+        expect(mail.subject).to eq "[wca-report] [Oceania] Comp of the Future 2016"
+        expect(mail.to).to eq ["reports@worldcubeassociation.org"]
+        expect(mail.cc).to match_array competition.delegates.pluck(:email) + ["disciplinary@worldcubeassociation.org"]
+        expect(mail.from).to eq ["notifications@worldcubeassociation.org"]
+        expect(mail.reply_to).to match_array competition.delegates.pluck(:email)
+      end
+
+      it "renders the body" do
+        expect(mail.body.encoded).not_to match(/@WRC/)
+        expect(mail.body.encoded).to match(/@WDC: Feedback requested on incidents: 4, 5, 6/)
+        expect(mail.body.encoded).to match(/This was a great competition/)
+      end
+    end
+
+    context "wrc feedback requested" do
+      before(:each) do
+        competition.delegate_report.update_attributes!(wrc_feedback_requested: true, wrc_incidents: "1, 2, 3")
+      end
+
+      it "renders the headers" do
+        expect(mail.subject).to eq "[wca-report] [Oceania] Comp of the Future 2016"
+        expect(mail.to).to eq ["reports@worldcubeassociation.org"]
+        expect(mail.cc).to match_array competition.delegates.pluck(:email) + ["regulations@worldcubeassociation.org"]
+        expect(mail.from).to eq ["notifications@worldcubeassociation.org"]
+        expect(mail.reply_to).to match_array competition.delegates.pluck(:email)
+      end
+
+      it "renders the body" do
+        expect(mail.body.encoded).to match(/@WRC: Feedback requested on incidents: 1, 2, 3/)
+        expect(mail.body.encoded).not_to match(/@WDC/)
+        expect(mail.body.encoded).to match(/This was a great competition/)
+      end
+    end
+
+    context "no wrc feedback" do
```

Comment:
I guess it's not that important, but there isn't a `"no wdc feedback"`.

Reply:
Oops... yeah this is a combo no wdc + no wrc feedback test (it checks for no "/@WDC/" in the body), just forgot to change the name. Fixed :)

Reply:
Thanks for addressing this!